bolt/spark/array.py

```diff
@@ -46,6 +46,19 @@ def unpersist(self):
         """
         self._rdd.unpersist()
 
+    def repartition(self, npartitions):
+        """
+        Repartitions the underlying RDD
+
+        Parameters
+        ----------
+        npartitions : int
+            Number of partitions to repartion the underlying RDD to
+        """
+
+        self._rdd.repartition(npartitions=npartitions)
```

Comment:
I don't think `repartition` doesn't work through a side-effect. You need to catch the repartitions array from the return value.

```python
self._rdd = self._rdd.repartition(npartitions=npartitions)
return self
```